Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/checkins/checkout/ContactJournalCheckInEntryCreator.kt

```diff
@@ -61,7 +61,7 @@ class ContactJournalCheckInEntryCreator @Inject constructor(
         // and Check-in EndDate and rounding it to the closest 15-minute duration
```

Comment:
can you pease update the comment here ? it is still mentioning 15 minutes

Reply:
Updated comment